chore(blog): add marketing q1 report (#3911)
asyncapi/website: markdown/blog/2025-Q1-marketing-report.md

---
title: "AsyncAPI Q1 Marketing WG Report"
date: 2025-04-01T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/marketing-images/marketing-q1-report.webp
authors:
  - name: Atinuke Oluwabamikemi Kayode
    photo: /img/avatars/bami.webp
    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
    byline: AsyncAPI Community Marketing Specialist

> excerpt: 'AsyncAPI's marketing efforts saw steady growth, increased engagement, and a stronger community presence. Looking ahead, we aim to amplify impact and expand outreach.'

Reviewer: ```suggestion
excerpt: "AsyncAPI's marketing efforts saw steady growth, increased engagement, and a stronger community presence. Looking ahead, we aim to amplify impact and expand outreach."
```

---
title: "AsyncAPI Q1 Marketing WG Report"

> date: 2025-04-01T06:00:00+01:00

Reviewer 2: Change the date to the 14th

type: Communication
tags:
  - Project Status
cover: /img/posts/marketing-images/marketing-q1-report.webp
authors:
  - name: Atinuke Oluwabamikemi Kayode
    photo: /img/avatars/bami.webp
    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
    byline: AsyncAPI Community Marketing Specialist
excerpt: "AsyncAPI's marketing efforts saw steady growth, increased engagement, and a stronger community presence. Looking ahead, we aim to amplify impact and expand outreach."
---

> AsyncAPI’s marketing efforts over the past three months have focused on expanding community reach, driving awareness for key initiatives, and solidifying our position in the API and open-source ecosystem.

Reviewer 3: ```suggestion
AsyncAPI’s marketing efforts over the past three months have been focused on expanding community reach, driving awareness for key initiatives, and solidifying our position in the API and open-source ecosystem.
```

> AsyncAPI’s marketing efforts over the past three months have been focused on expanding community reach, driving awareness for key initiatives, and solidifying our position in the API and open-source ecosystem.

Reviewer 2: ```suggestion
Over the past three months, AsyncAPI’s marketing efforts have focused on expanding community reach, driving awareness for key initiatives, and solidifying our position in the API and open-source ecosystem.
```

This report provides an overview of our marketing activities, including social media performance, content marketing, events, and future strategies to optimize our impact.

## Key Objectives

- Enhance brand awareness and positioning within the API and open-source space.
- Increase engagement across social media and community platforms.

> - Promote AsyncAPI-led events and initiatives, including Google Summer of Code (GSoC) and the AsyncAPI Confrences and Webinar.

Reviewer 3: ```suggestion
- Promote AsyncAPI-led events and initiatives, including Google Summer of Code (GSoC), the AsyncAPI Conferences and Webinars.
```

- Expand content marketing efforts to provide valuable industry insights and drive audience engagement.

## Marketing Performance Overview

### Social Media Performance

| **LinkedIn Metric**         | **Value**         |**last Quarter Value**|**% Change** |
|--------------------|-----------------------------|---------------|----------------|
|  Followers       | 4,255           |  3,949 | +8%         |
| New Followers        | 313         | 263  | +20%            |
| Posts       | 48        | 93 | -48%         |
|Impressions       |        23,715 | 29,366     | -18% |
|Clicks      |     1,495| 1398       |       +8% |
|Engagement Rate      |      7.52%     | 6.23% |  +21%    |

### Key Insights

- The 18% rise in new followers suggests growing interest in AsyncAPI’s content.
- Engagement rate increased by 13%, indicating that our content resonates well with our audience.

> - Click-through rates need improvement, signaling a need for stronger CTAs and content hooks.

Reviewer 2: ```suggestion
- Click-through rates need improvement, signaling stronger CTAs and content hooks.
```

| **X (Formerly Twitter)Metric**         | **Value**         |**% last Quarter** |**% Change** |
|--------------------|-----------------------------|----------------|-----------------|
|  Tweets      | 89         |      162   |-44%  |
| Retweets        | 24       |       54     |  -53% |
|Impressions       |         14,621|    27,823  |  -46% |
|Engagements     |       278  |   543      |  -48% |
|Replies     |      3  |  2     | +75% |
| Clicks       |    101       |     199      |  -48% |
|Likes          |       93   |        188      |  -49% |
|New Followers|         36     | -27| +239%|

### Key Insights

> Tweet output dipped by 44%, leading to a -46% dip in impressions as well.

Reviewer 2: ```suggestion
Tweet output dipped by 44%, leading to a -46% impression dip.
```

Replies saw a 75% increase, but overall reply numbers remain low.

> New followers count rose by +239%, given the dip and lose of followers in the previous quarter. 

Reviewer 2: ```suggestion
The number of new followers rose by +239%, given the dip and loss of followers in the previous quarter. 
```

> Overall, the decline in some metrics can be attributed to the higher number of events in the previous quarter, which naturally led to more content and activity compared to this quieter quarter.

Reviewer 2: ```suggestion
Overall, the decline in some metrics can be attributed to the higher number of events in the previous quarter, which naturally led to more content and activity than this quieter quarter.
```

### Website Performance 

<Figure
  src="/img/posts/marketing-images/goggle-analysis-page-view.webp"
   alt="A table showing AsyncAPI  Google Analytics Homepage performance from January to March"
  caption="AsyncAPI Google Analytics Homepage performance from January to March"
  className="text-center"
/>

**User Engagement Insights (Google Analytics)**
**Homepage Performance**

The main landing page, "AsyncAPI Initiative for event-driven APIs," received the highest traffic (17.3% of total views) but saw relatively low average time on page (17s), suggesting users may be skimming or using it as a jumping-off point.

**Strong Interest in Version 3.0.0**

The "3.0.0" documentation page held 16.6% of the total views with a solid 41s average time, showing deep interest in the newest spec release.

**High Engagement Pages**

> "AsyncAPI document structure" and "Payload schema" had fewer visits but strong time-on-page (44s and 36s respectively), indicating users are spending time understanding these technical sections.

Reviewer 2: ```suggestion
"AsyncAPI document structure" and "Payload schema" had fewer visits but strong time-on-page (44s and 36s, respectively), indicating users are spending time understanding these technical sections.
```

Similarly, "Understanding AsyncAPIs with a Practical Example" drew good engagement (28s avg.) and "Coming from OpenAPI" shows decent dwell time (30s), making them great learning pathway pages.

**Notable Drops**

"Generator" and "Tools" saw good traffic but experienced drops in average time spent (-13.4% and +1.9%, respectively). This might mean users don’t find these pages as intuitive or relevant and could benefit from clearer CTAs or improved content structure.

## Event & Community Engagement

### Google Summer of Code (GSoC) Mentorship Program

- AsyncAPI has successfully joined Google Summer of Code 2025, providing mentorship opportunities to aspiring open-source contributors.

> - [A dedicated informational webinar was held on March 14](https://www.youtube.com/watch?v=R7pl1F2dYLo), led by GSoC the Program Coordinator Elegbede (Ace) Azeez, to guide potential mentees.

Reviewer 3: ```suggestion
- [A dedicated informational webinar was held on March 14](https://www.youtube.com/watch?v=R7pl1F2dYLo), led by our GSoC Program Coordinator Elegbede (Ace) Azeez, to guide potential mentees.
```

> - [A dedicated informational webinar was held on March 14](https://www.youtube.com/watch?v=R7pl1F2dYLo), led by our GSoC Program Coordinator Elegbede (Ace) Azeez, to guide potential mentees.

Reviewer 2: ```suggestion
- [A dedicated informational webinar was held on March 14](https://www.youtube.com/watch?v=R7pl1F2dYLo), led by our GSoC Program Coordinator, Elegbede (Ace) Azeez, to guide potential mentees.
```

- Outcome: Increased awareness and participation in the AsyncAPI open-source ecosystem.

> ### Community Spotlight Initiative

Reviewer 2: It would be best to add the link to the blog

Author: Which blog?

This quarter, AsyncAPI launched the Community Spotlight Initiative to recognize outstanding contributors.
The initiative highlights individuals making significant contributions to AsyncAPI, whether through code, documentation, advocacy, or community building.

> The first spotlight is the Ashmit story. We look forward to having more stories in as we go.

Reviewer 2: redirect to the story

> ### AsyncAPI Conference 2025

Reviewer 2: Update the section with the current details and add links

- Speaker announcements for AsyncAPI Conference Singapore have generated positive engagement and anticipation.
- Promotional efforts included LinkedIn and Twitter posts, and community newsletters.
- Munich Call for Speaker is also out.

### Impact & Next Steps

- Increased community participation and AsyncAPI visibility.

> - Plan to host more live sessions, and developer Q&As to sustain momentum.

Reviewer 2: Here you can link to the previous live session, share the stats, and how you plan to sustain the momentum

  className="text-center"
/>

2. [Join us in welcoming Ankit Kumar, Head of Developer & Customer Success at @Aklivity, as a speaker at AsyncAPI Conference Singapore 2025! 🚀](https://www.linkedin.com/feed/update/urn:li:share:7308442490213126145/)

- Impressions: 1,332
- Likes: 54
- Comments: 1
- Clicks; 215
- Engagement Rate: 20.42%
- Key Takeaway: What worked well, thought leadership, community-driven 

<Figure
  src="/img/posts/marketing-images/third-best-performance-post.webp"
  alt="Announcement that Laurent Broudoux is stepping into a new role as an AsyncAPI Ambassador!"
  caption="A LinkedIn announcement showing the performance details of Laurent Broudoux post"
  className="text-center"
/>

3. [We’re thrilled to announce that Laurent Broudoux is stepping into a new role as an AsyncAPI Ambassador! 🚀](https://www.linkedin.com/feed/update/urn:li:share:7310201401454829568/)

- Impressions: 998
- Likes: 70
- Comments: 5
- Clicks: 49
- Engagement Rate: 12.73%
- Key Takeaway: It's community-driven, high-value content, visual appeal

## Future Marketing Strategy & Action Plan

### Key Focus Areas (Next 3 Months)

> - Expand LinkedIn, Manstadon, and Twitter audience growth with targeted engagement strategies.

Reviewer 2: ```suggestion
- Expand LinkedIn, Mastodon, and Twitter audience growth with targeted engagement strategies.
```

    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
    byline: AsyncAPI Community Marketing Specialist
excerpt: "AsyncAPI's marketing efforts saw steady growth, increased engagement, and a stronger community presence. Looking ahead, we aim to amplify impact and expand outreach."
---

Over the past three months, AsyncAPI’s marketing efforts have focused on expanding community reach, driving awareness for key initiatives, and solidifying our position in the API and open-source ecosystem.

This report provides an overview of our marketing activities, including social media performance, content marketing, events, and future strategies to optimize our impact.

## Key Objectives

- Enhance brand awareness and positioning within the API and open-source space.
- Increase engagement across social media and community platforms.
- Promote AsyncAPI-led events and initiatives, including Google Summer of Code (GSoC), the AsyncAPI Conferences and Webinars.
- Expand content marketing efforts to provide valuable industry insights and drive audience engagement.

## Marketing Working Group Performance Overview

### Social Media Performance

> | **LinkedIn Metric**         | **Value**         |**last Quarter Value**|**% Change** |

Reviewer 2: ```suggestion
| **LinkedIn Metric** | **Value** | **Last Quarter Value** | **% Change** |
```

- AsyncAPI has successfully joined Google Summer of Code 2025, providing mentorship opportunities to aspiring open-source contributors.
- [A dedicated informational webinar was held on March 14](https://www.youtube.com/watch?v=R7pl1F2dYLo), led by our GSoC Program Coordinator, Elegbede (Ace) Azeez, to guide potential mentees.
- Outcome: Increased awareness and participation in the AsyncAPI open-source ecosystem.

### Community Spotlight Initiative

This quarter, AsyncAPI launched the Community Spotlight Initiative to recognize outstanding contributors.
The initiative highlights individuals making significant contributions to AsyncAPI, whether through code, documentation, advocacy, or community building.
We published the first blog [Ashmit story](https://www.asyncapi.com/blog/community-spotlight-ashmit). This would be a monthly blog. [Find the community spotlight discussion on GitHub](https://github.com/asyncapi/community/issues/1732)

### AsyncAPI Conference 2025

- Speaker announcements for AsyncAPI Conference Singapore have generated positive engagement and anticipation.
- Promotional efforts included LinkedIn and Twitter posts, and community newsletters.

> - Munich Call for Speaker is also out.

Reviewer 2: Links to the CFP would be nice

Reviewer 2: And maybe add Lagos as well

Author: I can add it but It's not part of Q1.

> ### Impact & Next Steps

Reviewer 2: Reformat this section and make the bullet points align.

Author: It looks aligned at my end.

Or you are talking about something else?

<img width="1440" alt="Screenshot 2025-04-11 at 15 13 23" src="https://github.com/user-attachments/assets/ef04c470-ee6d-4568-85c2-f62aedc1871a" />

- Increased community participation and AsyncAPI visibility have been clear indicators of the positive impact of our recent efforts.

- Our first live session, featuring Eduardo Maldonado Fonseca Silva, marked the beginning of our "Monthly Webinar series: Learn With AsyncAPI" — an initiative designed to create accessible, developer-focused conversations around AsyncAPI concepts, tools, and real-world applications. Eduardo shared deep insights into event-driven architectures and how AsyncAPI boosts interoperability in modern systems. The session covered best practices, practical use cases, and how to get started with AsyncAPI. It concluded with an engaging Q&A where the community posed thoughtful, implementation-focused questions.

> [Watch on YouTube](https://www.youtube.com/watch?v=awfB_mwUp1c&t=268s)

Reviewer: ```suggestion
    <YouTube id="https://www.youtube.com/embed/awfB_mwUp1c" />
```

- To sustain and scale this initiative, we plan to:

  - Host monthly live sessions and developer Q&As featuring maintainers, contributors, and experts in the ecosystem.

  - Introduce community-driven topic nominations to ensure the sessions address current challenges and interests.

  - Repurpose session content into bite-sized clips, blog posts, and social media highlights for ongoing engagement.

  - Build a resource hub where past sessions and learning materials can be accessed anytime.

  - Collaborate with community ambassadors and contributors to co-host sessions, fostering ownership and inclusiveness.

### Top 3 Best-Performing Posts

These posts were selected based on their impressions and number of clicks.

<Figure
  src="/img/posts/marketing-images/best-performance-post.webp"
  alt="Line graph showing AsyncAPI LinkedIn follower growth increasing from 2,000 to 3,910 followers throughout 2024"
  caption="A LinkedIn announcement showing the performance details of AsyncAPI Technical Steering Committee (TSC) post"
  className="text-center"
/>

> 1. [We’re welcoming three new members to the AsyncAPI Technical Steering Committee (TSC)](https://www.linkedin.com/feed/update/urn:li:ugcPost:7287771093354893312/)
> 
> - Impressions: 2,470 Impressions
> - Likes: 67
> - Comments: 12
> - Clicks: 361
> - Engagement Rate: 18.18%
> - Key Takeaway: Why it performed well is because it's a community-driven and thought leadership post.
> 
> <Figure
>   src="/img/posts/marketing-images/second-best-performance-post.webp"
>   alt="Ankit Kumar, Head of Developer & Customer Success at @Aklivity, as a speaker at AsyncAPI Conference Singapore 2025"
>   caption="A LinkedIn announcement showing the performance details of Ankit Kumar speaker post"
>   className="text-center"
> />
> 
> 2. [Join us in welcoming Ankit Kumar, Head of Developer & Customer Success at @Aklivity, as a speaker at AsyncAPI Conference Singapore 2025! 🚀](https://www.linkedin.com/feed/update/urn:li:share:7308442490213126145/)
> 
> - Impressions: 1,332
> - Likes: 54
> - Comments: 1
> - Clicks; 215
> - Engagement Rate: 20.42%
> - Key Takeaway: What worked well, thought leadership, community-driven 
> 
> 
> <Figure
>   src="/img/posts/marketing-images/third-best-performance-post.webp"
>   alt="Announcement that Laurent Broudoux is stepping into a new role as an AsyncAPI Ambassador!"
>   caption="A LinkedIn announcement showing the performance details of Laurent Broudoux post"
>   className="text-center"
> />
> 
> 3. [We’re thrilled to announce that Laurent Broudoux is stepping into a new role as an AsyncAPI Ambassador! 🚀](https://www.linkedin.com/feed/update/urn:li:share:7310201401454829568/)
> 
> 
> - Impressions: 998
> - Likes: 70
> - Comments: 5
> - Clicks: 49
> - Engagement Rate: 12.73%
> - Key Takeaway: It's community-driven, high-value content, visual appeal
> 

Reviewer: ```suggestion
1. [We’re welcoming three new members to the AsyncAPI Technical Steering Committee (TSC)](https://www.linkedin.com/feed/update/urn:li:ugcPost:7287771093354893312/)

    - Impressions: 2,470 Impressions
    - Likes: 67
    - Comments: 12
    - Clicks: 361
    - Engagement Rate: 18.18%
    - Key Takeaway: Why it performed well is because it's a community-driven and thought leadership post.

    <Figure
    src="/img/posts/marketing-images/second-best-performance-post.webp"
    alt="Ankit Kumar, Head of Developer & Customer Success at @Aklivity, as a speaker at AsyncAPI Conference Singapore 2025"
    caption="A LinkedIn announcement showing the performance details of Ankit Kumar speaker post"
    className="text-center"
    />

2. [Join us in welcoming Ankit Kumar, Head of Developer & Customer Success at @Aklivity, as a speaker at AsyncAPI Conference Singapore 2025! 🚀](https://www.linkedin.com/feed/update/urn:li:share:7308442490213126145/)

    - Impressions: 1,332
    - Likes: 54
    - Comments: 1
    - Clicks; 215
    - Engagement Rate: 20.42%
    - Key Takeaway: What worked well, thought leadership, community-driven 

    <Figure
    src="/img/posts/marketing-images/third-best-performance-post.webp"
    alt="Announcement that Laurent Broudoux is stepping into a new role as an AsyncAPI Ambassador!"
    caption="A LinkedIn announcement showing the performance details of Laurent Broudoux post"
     className="text-center"
    />

3. [We’re thrilled to announce that Laurent Broudoux is stepping into a new role as an AsyncAPI Ambassador! 🚀](https://www.linkedin.com/feed/update/urn:li:share:7310201401454829568/)

    - Impressions: 998
    - Likes: 70
    - Comments: 5
    - Clicks: 49
    - Engagement Rate: 12.73%
    - Key Takeaway: It's community-driven, high-value content, visual appeal
```

## Future Marketing Strategy & Action Plan

### Key Focus Areas (Next 3 Months)

- Expand LinkedIn, Mastodon, and Twitter audience growth with targeted engagement strategies.
- Increase participation in live sessions.
- Strengthen the Community Spotlight Initiative by featuring more contributors and increasing visibility.
- Resume the AsyncAPI Student Ambassador Initiative documentation.
- Leverage event marketing to drive awareness and contribution to AsyncAPI initiatives.

> AsyncAPI’s marketing efforts over the past three months have led to steady growth in brand awareness, engagement, and community participation.

Reviewer 2: Is this a heading or part of the bullet points

Author: It's a new paragraph.
